packages/styled-components/src/components/primitives/Chat/Conversation/Message/Message.stories.tsx

```diff
@@ -48,14 +37,21 @@ export const Placements = () => (
 export const Colors = () => (
   <>
     <h1>Message colors</h1>
-    {Object.keys(Message.Colors).map((color: any) => (
+    {Object.keys(Message.BackgroundColors).map((color: any) => (
       <>
```

Comment:
1. Change `color: any to backgroundColor: any`
2. Instead of rendering a list of fragments, change these fragments for a div which will have a key property (you can add an index to the map as a value)

Reply:
done!